sys/net/network_layer/ng_ipv6/netif/ng_ipv6_netif.c

```diff
@@ -45,7 +48,7 @@ static char addr_str[NG_IPV6_ADDR_MAX_STR_LEN];
 #endif
 
 static ng_ipv6_addr_t *_add_addr_to_entry(ng_ipv6_netif_t *entry, const ng_ipv6_addr_t *addr,
-                                          uint8_t prefix_len, uint8_t flags)
+        uint8_t prefix_len, uint8_t flags)
```

Comment:
^^ Seems a bit inconsistent (for example with 334) to me, but that's me nitpicking...